protos/IncomingPushMessageSignal.proto

```diff
@@ -96,30 +97,47 @@ message SyncMessage {
 
   message Request {
     enum Type {
-      UNKNOWN     = 0;
-      CONTACTS    = 1;
-      GROUPS      = 2;
-      BLOCKED     = 3;
+      UNKNOWN  = 0;
+      CONTACTS = 1;
+      GROUPS   = 2;
+      BLOCKED  = 3;
     }
+
     optional Type type = 1;
   }
+
   message Read {
     optional string sender    = 1;
     optional uint64 timestamp = 2;
   }
 
-  optional Sent     sent     = 1;
-  optional Contacts contacts = 2;
-  optional Groups   groups   = 3;
-  optional Request  request  = 4;
-  repeated Read     read     = 5;
-  optional Blocked  blocked  = 6;
+  message Verification {
+    enum State {
+      DEFAULT            = 0;
+      VERIFIED           = 1;
+      NO_LONGER_VERIFIED = 2;
+    }
+
+    optional State  state       = 1;
+    // The e164 phone number of the user.
+    optional string destination = 2;
+    optional bytes  identityKey = 3;
+  }
+
+  optional Sent     sent             = 1;
+  optional Contacts contacts         = 2;
+  optional Groups   groups           = 3;
+  optional Request  request          = 4;
+  repeated Read     read             = 5;
+  optional Blocked  blocked          = 6;
+  repeated Verification verification = 7;
```

Comment:
Interesting that this is `repeated`. I guess that's so we can send a single sync message when dismissing a banner or warning in a group?